src/main/resources/reference.conf

```diff
@@ -242,6 +255,19 @@ cassandra-snapshot-store {
   # Port of contact points in the cluster.
   # Will be ignored if the contact point list is defined by host:port pairs.
   port = 9042
+  
+  # An implementation of akka.persistence.cassandra.ContactPointsProvider
+  # can be used to lookup the contact points of the Cassandra cluster
+  # asynchronously instead of given them in the configuration. If this
+  # is specified the contact-points and port properties will be ignored.
+  # Define fully qualified class name of the implementation of the 
+  # akka.persistence.cassandra.ContactPointsProvider. It may optionally
+  # have a constructor with a ActorSystem parameter.
+  contact-points-provider = ""
+  
+  # The identifier that will be passed as parameter to the
+  # ContactPointsProvider.lookup method. 
+  contact-points-provider-id = ""
```

Comment:
Do you think that this approach is better than providing an actor selection path (as per the original example I provided)? Using a selection path allows you to then reference an actor that may already exist for the purposes of service location, rather than this module be responsible for creating one.